README.md

```diff
@@ -110,74 +108,55 @@ Note on AIX Support:
 
+```ruby
+default['audit']['reporter'] = 'chef-server-compliance'
 
-node.default['audit']['profiles'].push("path": "#{PROFILES_PATH}/mylinux-failure-success")
+# Omit this to use the latest InSpec
+default['audit']['inspec-version'] = '1.29.0'
 
-```ruby
-"audit": {
-  "reporter": "chef-server-compliance",
-  "inspec_version": "1.29.0",
-  "profiles": [
-    # profile from Chef Compliance
+default['audit']['profiles'].push(
+    # Profile from Chef Compliance
     {
-      "name": "linux",
-      "compliance": "base/linux"
+      'name': 'linux',
+      'compliance': 'base/linux'
     },
-    # profile from Chef Compliance at a particular version
+    # Profile from Chef Compliance at a particular version
     {
-      "name": "linux-baseline",
-      "compliance": "user/linux-baseline",
-      "version": "2.1.0"
+      'name': 'linux-baseline',
+      'compliance': 'user/linux-baseline',
+      'version': '2.1.0'
     },
-    # profile from supermarket
+    # Profile from Supermarket
     # note: If reporting to Compliance, the Supermarket profile needs to be uploaded to Chef Compliance first
+    # note: Artifactory's Supermarket implementation—"Chef Cookbook repository"—does not support InSpec compliance profiles at this time
     {
-      "name": "ssh",
-      "supermarket": "hardening/ssh-hardening"
+      'name': 'ssh',
+      'supermarket': 'hardening/ssh-hardening'
     },
-    # local Windows path
+    # Profile from local Windows path
     {
-      "name": "brewinc/win2012_audit",
+      'name': 'brewinc/win2012_audit',
       # filesystem path
-      "path": "E:/profiles/win2012_audit"
+      'path': 'E:/profiles/win2012_audit'
     },
-    # github
+    # Profile from GitHub
     {
-      "name": "ssl",
-      "git": "https://github.com/dev-sec/ssl-benchmark.git"
+      'name': 'ssl',
+      'git': 'https://github.com/dev-sec/ssl-benchmark.git'
     },
-    # url
+    # Profile from URL
     {
-      "name": "ssh",
-      "url": "https://github.com/dev-sec/tests-ssh-hardening/archive/master.zip"
+      'name': 'ssh',
+      'url': 'https://github.com/dev-sec/tests-ssh-hardening/archive/master.zip'
     }
-  ]
-}
-```
-
-You can also configure in a policyfile like this:
-
-```ruby
-default["audit"] = {
-  "reporter" => "chef-server-compliance",
-  "profiles" => [
-    {
-      "name": "linux",
-      "compliance": "base/linux"
-    },
-    {
-      "name": "ssh",
-      "compliance": "base/ssh"
-    }
-  ]
-}
+)
 ```
 
 #### Attributes
 
-You can also pass in [InSpec Attributes](https://www.inspec.io/docs/reference/profiles/) to your audit run. You do this by defining the attributes here:
+You can also pass in [InSpec Attributes](https://www.inspec.io/docs/reference/profiles/) to your audit run. You do this by defining the attributes like so:
```

Comment:
You can also pass InSpec Attributes into your audit run. Do this by defining attributes: